www/src/utils/sidebar/get-active-item-parents.js

```diff
@@ -15,7 +15,7 @@ const isParentActive = (sections, parentTitle) => {
 const getActiveItemParents = (itemList, activeItemLink, activeItemParents) => {
   if (activeItemLink.parentTitle) {
     const bar = isParentActive(itemList, activeItemLink.parentTitle)
-    activeItemParents.push(bar.title)
+    activeItemParents.push(bar)
```

Comment:
Lol at `bar` (that was me) 🤦‍♂ … naming things is hard, but this one was lazy-stupid.